avocado/core/loader.py

```diff
@@ -44,6 +44,41 @@
 ALL = True
 
 
+def filter_test_tags(test_suite, filter_by_tags):
+    '''
+    The filtering mechanism is limited to instrumented tests with tags set
+    '''
```

Comment:
Would you please use `"""` instead of `'''`? It allows one to comment out even commented code by  using `'''`.